plugins/modules/fusion_info.py

```diff
@@ -491,17 +490,17 @@ def warning_argument_none(name, requirement):
         "role_assignments": role_assignments_num,
         "tenant_spaces": tenant_spaces_num,
         "volumes": volumes_num,
-        "placements_groups": placements_groups_num,
+        "placement_groups": placement_groups_num,
         "snapshots": snapshots_num,
         "availability_zones": availability_zones_num,
-        "appliances": appliances_num,
```

Comment:
Don't we need to leave this argument until v2.0 and print warning that it will be deprecated?

Reply:
Already did the same kind of change in https://github.com/Pure-Storage-Ansible/Fusion-Collection/pull/110 and https://github.com/Pure-Storage-Ansible/Fusion-Collection/pull/112